prow/spyglass/lenses/buildlog/lens.go

```diff
@@ -198,8 +220,8 @@ func (lens Lens) Body(artifacts []api.Artifact, resourceDir string, data string,
 		}
 		artifact := av.ArtifactName
 		meta, _ := a.Metadata()
-		start, end := -1, -1
 
+		start, end := -1, -1
```

Comment:
Is this diff spurious? Or is there something here I am missing? :grimacing:

Reply:
Done